addons/wiki/static/ShareJSDoc.js

```diff
@@ -180,6 +180,8 @@ var ShareJSDoc = function(url, metadata, viewText, editor) {
         onopen(event);
         if (madeConnection) {
             viewModel.status('connected');
+        } else {
+            viewModel.status('disconnect');
```

Comment:
Shouldn't this be 'disconnected' rather than 'disconnect'?

Reply:
Yes, for consistencies sake, but any status other than connecting or connected is going to be read as disconnected